### PR TITLE
Us124 apply prototype styling

### PR DESCRIPTION
**Pull request by danielmarker**
Matching of the prototype styling and bringing code up to production ready.

### CODE REVIEW

**Comment by danielmarker** [comment]
@Kirosoft Can we discuss this on Thursday please?

File: application/config.py [hunk 1]
```diff
@@ -4,4 +4,4 @@
 
 
 DEED_API_BASE_HOST = os.getenv('DEED_API_ADDRESS',
-                               'http://deed-api.dev.service.gov.uk')
+                               'http://10.10.10.10:9020')
```
Comment (tsangsimon) [inline application/config.py]: Why was this changed to a localhost address? In production we should have the default pointing to whatever the LIVE address is.

Comment (danielmarker) [inline application/config.py]: Fixed

File: application/templates/viewdeed.html [hunk 2]
```diff
@@ -5,25 +5,95 @@
...
+        <div class="text">
+            Lender Information will go here
+        </div>
+
+        <h2 class="heading-medium">Borrower</h2>
+        <div class="grid-row">
+            {% for borrower in deed_data.deed.borrowers %}
+            <div class="column-third">
+              <p>
+                <b><a>{{ borrower.forename}} {{ borrower.middle_name}}{% if borrower.middle_name %} {% endif %}{{
```
Comment (endamccormack) [inline application/templates/viewdeed.html]: its incorrect to use an anchor tag here

File: application/templates/viewdeed.html [hunk 3]
```diff
@@ -5,25 +5,95 @@
...
+
+    <div class="text">
+      <div class="alert-inline">
+        <p><b>This is a legal document &mdash; please check all details carefully.</b></p>
+      </div>
+    </div>
+    <div class="column-two-thirds">
+    <article class="deed-document--bounds">
+        <h2 class="heading-large">Electronic Mortgage Deed</h2>
+            <h2 class="heading-medium">Property: </h2>
```
Comment (endamccormack) [inline application/templates/viewdeed.html]: why did you use a h2, the logical progression would be a h3.

File: application/templates/viewdeed.html [hunk 4]
```diff
@@ -5,25 +5,95 @@
...
+                <span itemprop="postalCode">PO57 CDE</span>
+              </p>
+
+            <dl class="definition-inline">
+            <dt><b>Title number</b></dt>
+                <dd>{{ deed_data.deed.title_number }}</dd>
+            </dl>
+
+          <h2 class="heading-medium">Lender</h2>
+        <div class="text">
```
Comment (endamccormack) [inline application/templates/viewdeed.html:32]: indentation

File: application/templates/viewdeed.html [hunk 5]
```diff
@@ -5,25 +5,95 @@
...
+              <p>
+                <b><a>{{ borrower.forename}} {{ borrower.middle_name}}{% if borrower.middle_name %} {% endif %}{{
+                borrower.surname}}</a></b><br>
+              </p>
+            </div>
+            {% endfor %}
+        </div>
+
+        <div class="text">
+
```
Comment (endamccormack) [inline application/templates/viewdeed.html]: tidier without this newline in my opinion

File: application/templates/viewdeed.html [hunk 6]
```diff
@@ -5,25 +5,95 @@
...
+              <li>
+                <p>The lender is obliged to make further advances and applies for the obligation to be entered in the register.</p>
+              </li>
+              <li>
+                <p>The borrower applies to enter a restriction in the register that no disposition of the registered estate by the proprietor of the registered estate is to be registered without a written consent signed by the proprietor for the time being of the charge dated [the date of this charge] in favour of Bank of England Plc referred to in the charges register.</p>
+              </li>
+            </ol>
+        </div>
+        <p class="text">
+
```
Comment (endamccormack) [inline application/templates/viewdeed.html:65]: probably better with a new line after the div rather then after the p

File: application/templates/viewdeed.html [hunk 7]
```diff
@@ -5,25 +5,95 @@
...
+              <label class="heading-small" for="auth-code">Effective date and time</label></br>
+              <p >This charge takes effect when the registrar receives notification from
+                  Pure Law that the charge is to take effect.</p>
+              <p class="bordered-output-not-present">[Effective date and time will be shown here]</p>
+              <label class="heading-small" for="signature">Borrower's electronic signature(s)</label><br>
+            {% for borrower in deed_data.deed.borrowers %}
+              <p class="bordered-output-not-present" id="signature{{loop.index}}">[e-signature of {{ borrower.forename}} {{ borrower.middle_name}}{% if borrower.middle_name %} {% endif %}{{
+                borrower.surname}} goes here]</p>
+            {% endfor %}
+                    <div class="form-group">
```
Comment (endamccormack) [inline application/templates/viewdeed.html]: indentation

File: application/templates/viewdeed.html [hunk 8]
```diff
@@ -5,25 +5,95 @@
     {{ super() }}
     <div>&nbsp;</div>
```
Comment (endamccormack) [inline application/templates/viewdeed.html]: remove this

File: application/static/landregistry/assets/stylesheets/application.scss [hunk 9]
```diff
@@ -22,4 +22,19 @@ float: left;
 .verfied-bullet-numbers-fa {
   font-family: 'nta';
   color:white;
-}
\ No newline at end of file
+}
+
```
Comment (endamccormack) [inline application/static/landregistry/assets/stylesheets/application.scss:26]: new line not needed.

**Comment by endamccormack** [comment]
- Few things need to be removed
- HTML indentation is pretty poor all over the place which is making it pretty hard to read
- font awesome seems to be a generated file so there is no need to add the css file

File: application/static/landregistry/assets/stylesheets/application.scss [hunk 10]
```diff
@@ -22,4 +22,19 @@ float: left;
...
+
+.bordered-output-present{
+    border: 1px solid #bfc1c3;
```
Comment (andymantell) [inline application/static/landregistry/assets/stylesheets/application.scss]: These new lines appear to be indented with 4 spaces compared to the 2 used elsewhere. Would be worth configuring your editor to match the surrounding code if it can do that

File: application/static/landregistry/assets/stylesheets/landregistry.scss [hunk 11]
```diff
@@ -161,4 +161,18 @@ dl {
   dd {
     float: left;
   }
-}
\ No newline at end of file
+}
+
+.deed-document--bounds {
```
Comment (andymantell) [inline application/static/landregistry/assets/stylesheets/landregistry.scss]: As much as I like BEM (I do!), none of the surrounding code is written using BEM conventions so I wonder if these should perhaps be `.deed-document-bounds` and `.deed-document-divider`?

File: application/templates/viewdeed.html [hunk 12]
```diff
@@ -5,25 +5,95 @@
...
+                <p>The lender is obliged to make further advances and applies for the obligation to be entered in the register.</p>
+              </li>
+              <li>
+                <p>The borrower applies to enter a restriction in the register that no disposition of the registered estate by the proprietor of the registered estate is to be registered without a written consent signed by the proprietor for the time being of the charge dated [the date of this charge] in favour of Bank of England Plc referred to in the charges register.</p>
+              </li>
+            </ol>
+        </div>
+        <p class="text">
+
+              <label class="heading-small" for="auth-code">Effective date and time</label></br>
```
Comment (andymantell) [inline application/templates/viewdeed.html]: Are these labels going to have something associated with them later? The for attributes don't relate to any input fields...

File: application/templates/viewdeed.html [hunk 13]
```diff
@@ -5,25 +5,95 @@
...
+                        <p class="bordered-output-not-present" id="registrarsig">[Will be applied when the charge takes effect]</p>
+                    </div>
+            <div class="text">
+              <label>E-MD ref: {{deed_data.deed.md_ref}}</label><br>
+
+            </div>
+        </p>
+    </article>
+    <div class="text">
+        <form action="/searchdeed">
```
Comment (andymantell) [inline application/templates/viewdeed.html]: Are there more things expected here later? Why is this a form?

**Comment by endamccormack** [comment]
:+1:

**Comment by tsangsimon** [comment]
:+1:
